docs/source/performance.md

```diff
@@ -76,11 +73,11 @@ peer:
             gatewayService: 500
 ```
 
-The peer gateway service first released in V2.4 of Hyperledger Fabric introduced a limit in the `gatewayService` with a default of 500. However, this may restrict the TPS of a network so you may need to increase this value to allow more concurrent requests. Good results have been seen with 20000 concurrent requests, higher values may be counter-productive.
+The Peer Gateway Service, first released in v2.4 of Hyperledger Fabric, introduced the `gatewayService` limit with a default of 500. However, this default can restrict network TPS, so you may need to increase this value to allow more concurrent requests. The maximum suggested limit is 20000 concurrent requests.
```

Comment:
`The maximum suggested limit is 20000 concurrent requests`. This is not a suggested limit. Actually it links to some performance results which are now not part of the overall document. I think we should now remove this sentence

Reply:
ok